redist/org.goldendict.GoldenDict.appdata.xml

```diff
@@ -34,6 +34,10 @@
   </screenshots>
   <url type="homepage">http://goldendict.org</url>
   <update_contact>https://github.com/goldendict/goldendict</update_contact>
-  <launchable type="desktop-id">goldendict.desktop</launchable>
+  <launchable type="desktop-id">org.goldendict.GoldenDict.desktop</launchable>
   <content_rating type="oars-1.0" />
+  <provides>
+    <id>org.goldendict.desktop</id>
+    <id>goldendict</id>
```

Comment:
Do you think that `<id>goldendict</id>` in `<provides>` is necessary because of the previous name of this file, which was inconsistent with `<id>org.goldendict.desktop</id>`?

Reply:
That was the id of the application. The filename didn't have to be the same as the id, although that's not standard and causes confusion. 
The reason I add `<id>goldendict</id>` is that I'm not sure how the packaging was done. But it looks like fedora and archlinux simply copies the app metadata file which preserves the id defined in the file. So I'll just remove the `<id>goldendict</id>`

Reply:
Yes, I think there is no need to add an extra `provides` entry since your other changes don't affect a hypothetical already broken assumption of `id=goldendict`.

Reply:
And that's removed. Any other missing parts?